test/sql/user.sql

```diff
@@ -1,4 +1,4 @@
-CREATE USER 'root'@'%' IDENTIFIED BY 'root';
+-- CREATE USER 'root'@'%' IDENTIFIED BY 'root';
```

Comment:
Finally, this did the trick :)

Reply:
> Finally, this did the trick :)

What are your thoughts on doing `CREATE USER IF NOT EXISTS ...` rather than commenting?